src/vs/workbench/contrib/issue/browser/issueQuickAccess.ts

```diff
@@ -18,55 +23,119 @@ export class IssueQuickAccess extends PickerQuickAccessProvider<IPickerQuickAcce
 	constructor(
 		@IMenuService private readonly menuService: IMenuService,
 		@IContextKeyService private readonly contextKeyService: IContextKeyService,
-		@ICommandService private readonly commandService: ICommandService
+		@ICommandService private readonly commandService: ICommandService,
+		@IExtensionService private readonly extensionService: IExtensionService,
+		@IConfigurationService private readonly configurationService: IConfigurationService
 	) {
 		super(IssueQuickAccess.PREFIX, { canAcceptInBackground: true });
 	}
 
 	protected override _getPicks(filter: string): Picks<IPickerQuickAccessItem> | FastAndSlowPicks<IPickerQuickAccessItem> | Promise<Picks<IPickerQuickAccessItem> | FastAndSlowPicks<IPickerQuickAccessItem>> | null {
+		if (this.configurationService.getValue<string>('extensions.experimental.quickAcessExtensions')) {
```

Comment:
```suggestion
		if (!this.configurationService.getValue<boolean>('extensions.experimental.quickAcessExtensions')) {
		  return
		}
```

less indentation, easier to remove later